solidity/dashboard/src/components/Modal.jsx

```diff
@@ -114,6 +140,26 @@ export const ModalContextProvider = ({ children }) => {
     [openModal, onSubmitConfirmationModal, closeModal]
   )
 
+  const customModalWindowForMobileUsersClose = () => {
+    setModalWindowForMobileUsersDisplayed(true)
+    closeModal()
+  }
+
+  if (
+    showModalWindowForMobileUsers &&
+    !isOpen &&
+    !modalWindowForMobileUsersDisplayed
+  ) {
+    openModal(
+      <MobileUsersModal closeModal={customModalWindowForMobileUsersClose} />,
+      {
+        closeModal: customModalWindowForMobileUsersClose,
+        hideTitleBar: true,
+        isRounded: true,
+      }
+    )
+  }
+
```

Comment:
It should be wrapped by `useEffect` shouldn't it? Otherwise, if the width of the window will change and be greater than a threshold (in our case`1000`), the modal is still displayed.

Reply:
I fixed it but I'm also wondering if we should display the modal more than once. Right now we display it exactly once, so for example when we resize the window below 1000 (modal will appear), then resize again to >1000px (modal will disappear) and then again resize it back to <1000px, then the modal will not appear again. I think we don't need to display it many times and one is enough (until refresh) but I'm open to suggestions.

Reply:
I would prefer to display a modal each time the window resizes. But I will leave the final decision to @liz-shinn.

Reply:
Hmm 🤔 Let's discuss on the standup call. In the scenario Michal describes it would make sense to show the modal again when the window re-sizes. But, I'm wondering if trying to show it more than once will result in it showing over and over on refreshes, or navigating to a different page?

Reply:
Note: we discussed it on the standup call and decided to keep it like it is now, which means displaying modal only once (unless user refreshes)